### ISSUE TITLE
manual zoom buttons

### ISSUE THREAD

**Issue by 250602110504** [marmelab/EventDrops#268]
**Problem:**
I would like to know if it is possible to create buttons that let the user zoom and scroll like a mouse? I have tried a lot of different methods and no one works. 
I really hope someone could help me, because I have spend a lot of time on it, but still run into trouble. 

**Environment**
* EventDrops version: 1.3.0
* Browser: Chrome

Thanks for your time :)

**Comment by djhi** [comment]
Hi, and thanks for your question. As explained in the [react-admin contributing guide](https://github.com/marmelab/react-admin/blob/master/README.md#contributing), the right place to ask a "How To" question, get usage advice, or troubleshoot your own code, is [StackOverFlow](http://stackoverflow.com/questions/tagged/react-admin).

This makes your question easy to find by the core team, and the developer community. Unlike Github, StackOverFlow has great SEO, gamification, voting, and reputation. That's why we chose it, and decided to keep GitHub issues only for bugs and feature requests.

So I'm closing this issue, and inviting you to ask your question at:

http://stackoverflow.com/questions/tagged/react-admin

And once you get a response, please continue to hang out on the react-admin channel in StackOverflow. That way, you can help newcomers and share your expertise!

**Comment by 250602110504** [comment]
I just added this problem in addition to add a new feature to the project.

**Comment by Busteren** [comment]
Why do you mean by zoom button? The chart allows for zoom as you can see in the gif in the README.
Did you mean programmatically zoom by clicking a button? If so I am currently writing a PR to add that.

**Comment by 250602110504** [comment]
Yes, that's what I exactly mean. Programatically zoom by clicking a button. 

Thanks for your answer.

**Comment by Busteren** [comment]
Okay, I might have a PR ready in the next day or two. I don't think it will be included in a release just yet, however, you can always incorporate the changes yourself.
